users/tests/test_views.py

```diff
@@ -156,3 +156,56 @@ def test_user_update_view_should_not_update_user_on_post_if_form_is_invalid(self
         self.user.refresh_from_db()
         self.assertEqual(response.status_code, 200)
         self.assertEqual(phone_number_before_request, self.user.phone_number)
+
+
+class UserUpdateByAdminTests(TestCase):
+    def setUp(self):
+        self.user = CustomUser(
+            email="testuser@codepoets.it", password="newuserpasswd", first_name="John", last_name="Doe", country="PL"
+        )
+        self.user.full_clean()
+        self.user.save()
+        self.user_admin = AdminUserFactory()
+        self.client.force_login(self.user_admin)
+        self.user.refresh_from_db()
+
+    def test_user_update_by_admin_view_should_display_user_details_on_get(self):
+        response = self.client.get(path=reverse("custom-user-update-by-admin", args=(self.user.pk,)))
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, self.user.email)
+        self.assertContains(response, self.user.first_name)
+        self.assertContains(response, self.user.last_name)
```

Comment:
You can also add check like `self.assertTemplateUsed('template_name.html')`

Reply:
:+1: